File: JavaScript/1470. Shuffle the Array.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
    var a = [], b = nums.length;
    var t = 0;
    for(var i = 0;i < b;i+=2){
        a[i] = nums[t];
        a[i+1] = nums[t+n];
        t++;
    }
    return a;
};

File: Java/1470. Shuffle the Array.java
class Solution {
    public int[] shuffle(int[] nums, int n) {
        int[] r = new int[n * 2];
        for(int i = 0; i < n * 2; i += 2){
            r[i] = nums[i / 2];
            r[i + 1] = nums[i / 2 + n];
        }
        return r;
    }
}
